opentelemetry-zipkin/src/exporter/mod.rs

```diff
@@ -225,8 +146,7 @@ async fn zipkin_export(
     uploader.upload(zipkin_spans).await
 }
 
-#[async_trait]
-impl trace::SpanExporter for Exporter {
+impl trace::SpanExporter for ZipkinExporter {
```

Comment:
removing async_trait is welcome change, please follow the similar thing as done in https://github.com/open-telemetry/opentelemetry-rust/pull/2374

Reply:
there is no code change needed it was just redundant. it is similar to #2339